webrtc-stats.html

```diff
@@ -2056,90 +2211,113 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Only present for outbound audio tracks. The total number of audio samples that
-                  have been sent for this track.
+                  The total number of audio samples that have been sent for this track.
                 </p>
               </dd>
+            </dl>
+          </section>
+        </div>
+      </section>
+      <section id="raststats-dict*">
+        <h3>
+          <dfn>RTCRemoteAudioStreamTrackStats</dfn> dictionary
+        </h3>
+        <p>
+          An RTCRemoteAudioStreamTrackStats object represents the stats about one
+          audio MediaStreamTrack coming from a PeerConnection object for which one calls getStats.
+        </p>
+        <p>
+          It appears in the stats as soon as it is created on an RTCRtpReceiver object.
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCRemoteAudioStreamTrackStats : RTCAudioStreamTrackStats {
+             DOMHighResTimeStamp estimatedPlayoutTimestamp;
+             double              jitterBufferDelay;
```

Comment:
remote audio: jitterBufferEmittedCount should be listed here (it's already listed in the below member definitions).

Reply:
Thanks, I caught this in the rebase of https://github.com/w3c/webrtc-stats/pull/273.